tests/test-cases/Group23-VIC-Machine-Service/23-02-VCH-List.md

```diff
@@ -10,17 +10,72 @@ To verify vic-machine-server can return a list of VCHs including the expected in
 # Environment:
 This test requires a vSphere system where VCHs can be deployed 
 
-# Test Steps:
-1. Deploy a VCH
-2. Get a list of all VCHs
-3. Get a list of all VCHs in the test datacenter
-4. Attempt to list all VCHs in an invalid datacenter
-5. Attempt to list all VCHs in an invalid compute resource
-6. Attempt to list all VCHs in an invalid datacenter and compute resource
-
-# Expected Outcome:
-* The results of 2-3 should contain the VCH created in 1, with the correct ID.
-* The requests in 4-6 should result in an appropriate error message.
-
-# Possible Problems:
-None known
+**Note:** Multiple versions of tests must be implemented:
+ * With and without datacenter-scoped URLs.
+ * Using both username/password- and session-based authentication.
+
+It should not be necessary to implement all four combinations for every test case, but full coverage should be provided for at least one of the basic test cases.
+
+
+Basic Validation
+----------------
+
+These tests are intended to verify the feature at a basic level.
+
+###  1. Without creating any VCHs, verify that an empty list is properly returned
+
+(Not yet implemented)
+
+###  2. Create a VCH and verify it is returned in the list, with correct information
+
+###  3. Create two VCHs and verify both are returned in the list, with correct information for each
+
+(Not yet implemented)
+
+###  4. Create 64 VCHs and verify all are returned in the list, with correct information for each
+
+(Not yet implemented)
+
```

Comment:
Do we want to have a test case for listing VCH power state? An example would be, `create three VCHs, power off one of them, list and see if correct power states are returned`